sql/updates/world/2016_01_XX_XX_world.sql

```diff
@@ -0,0 +1,26 @@
+-- THERE ARE A LOT OF EXCESS COMMENTS IN THIS FILE. They will be removed after code review.
+-- Add sniffed gossip texts to menu
```

Comment:
that's possible but nothing found in my sniff, assign the higher broadcasttext id to the gossip of 20235